app/src/main/java/com/practicum/playlistmaker/search/data/db/dao/PlaylistDao.kt

```diff
@@ -0,0 +1,25 @@
+package com.practicum.playlistmaker.search.data.db.dao
+
+import androidx.room.Dao
+import androidx.room.Insert
+import androidx.room.Query
+import androidx.room.Update
+import com.practicum.playlistmaker.search.data.db.entities.PlaylistEntity
+import kotlinx.coroutines.flow.Flow
+
+@Dao
+interface PlaylistDao {
+
+    @Insert()
```

Comment:
```suggestion
    @Insert
```
немного красивее будет